pkg/testutils/sqlutils/sql_runner.go

```diff
@@ -135,9 +135,29 @@ func (sr *SQLRunner) QueryStr(query string, args ...interface{}) [][]string {
 // CheckQueryResults checks that the rows returned by a query match the expected
 // response.
 func (sr *SQLRunner) CheckQueryResults(query string, expected [][]string) {
-	res := sr.QueryStr(query)
+	res := sr.QueryResults(query)
 	if !reflect.DeepEqual(res, expected) {
 		file, line, _ := caller.Lookup(1)
 		sr.Errorf("%s:%d query '%s': expected:\n%v\ngot:%v\n", file, line, query, expected, res)
 	}
 }
+
+// QueryInt runs a query that returns a single int (or kills test on error).
+func (sr *SQLRunner) QueryInt(query string, args ...interface{}) int64 {
+	var i int64
+	if err := sr.DB.QueryRow(query, args...).Scan(&i); err != nil {
+		file, line, _ := caller.Lookup(1)
```

Comment:
is it time to override `{Fatal,Error}{,f}` to do this (with `caller.Lookup(2)`) instead of peppering it around?